### PR TITLE
Improve inventory table UI

### LINKED ISSUE #5280: bug fix - UI glitch in option on service Inventory options

### PR DESCRIPTION
**Pull request by LukasStordeur**
# Description

- Removed the Attribute Summary column.
- Modified the text to drilldown in the actions menu
- Improved calculation for the menu height for the drilldown. 
- Made sure the Deploy Progress column had enough width, made a switch case if we need to finetune the width of other columns. 
- Updated E2E tests for flake in Scenario 3. 

Closes #5280  and #5278 and #4934

https://github.com/inmanta/web-console/assets/44098050/152672e9-3f42-4644-aed2-96fb09a50a75

### CODE REVIEW

File: src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx [hunk 1]
```diff
@@ -257,7 +261,7 @@ export const RowActions: React.FunctionComponent<InstanceActionsProps> = ({
               </DrilldownMenu>
             }
           >
-            More options
+            More actions
```
Comment (matborowczyk) [inline src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx:264]: shouldn't that be in words file?

**Review by matborowczyk — COMMENTED** [review]
There is only minor thing with label hard-coded, and possibly that label could be taken to the test file as well, but that's optional

File: cypress/e2e/scenario-5-compile-reports.cy.js [hunk 2]
```diff
@@ -314,7 +314,7 @@ describe("5 Compile reports", () => {
       cy.get('[aria-label="row actions toggle"]', { timeout: 60000 })
         .eq(0)
         .click();
-      cy.get(".pf-v5-c-menu__item").contains("More options").click();
+      cy.get(".pf-v5-c-menu__item").contains("More actions").click();
```
[the remark this answers is not in the document]
Comment (LukasStordeur) [reply cypress/e2e/scenario-5-compile-reports.cy.js:317]: We never fetch from the word file in the E2E :)

[the remark this answers is not in the document]
Comment (matborowczyk) [reply cypress/e2e/scenario-5-compile-reports.cy.js:317]: Oh right, i overlooked that it is e2e file

**Review by matborowczyk — APPROVED** [review]

**Pull request closed without merging**